localisation/00_wc_sad.csv

```diff
@@ -749,7 +749,21 @@ RULE_STATIC_PORTRAITS_ENABLED;Enabled;;Aktiviert;;;;;;;;;;;x
 RULE_STATIC_PORTRAITS_ENABLED_DESC;Static portraits for certain lore characters are �Genabled�!.;;Statische Portraits f�r bestimmte historische Charaktere sind �Gaktiviert�!.;;;;;;;;;;;x
 RULE_STATIC_PORTRAITS_DISABLED;Disabled;;Deaktiviert;;;;;;;;;;;x
 RULE_STATIC_PORTRAITS_DISABLED_DESC;Static portraits for certain lore characters are �Rdisabled�!.;;Statische Portraits f�r bestimmte historische Charaktere sind �Rdeaktiviert�!.;;;;;;;;;;;x
-;;;;;;;;;;;;;;
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
 ;;;;;;;;;;;;;;x
+writing_whispers;Hears Whispers;;;;;;;;;;;;;x
+writing_whispers_desc;This character hears whispers of the Old Gods.;;;;;;;;;;;;;x
```

Comment:
```suggestion
writing_whispers_desc;This character hears the whispers of the Old Gods.;;;;;;;;;;;;;x
```

Reply:
Not sure, if we have to be specific hear. Without "the" it's just any whispers of the Old Gods, but nothing specific. I'm ok with both versions.

Reply:
Valid point, yeah. Both work, but the first is more vague. The second sounds like the Old Gods are whispering to the character